### PR TITLE
[UWP] Fix RadioButton disposing

### PR DESCRIPTION
**Pull request by yurkinh**
### Description of Change ###

Added Dispose method with events unsubscription 

### Issues Resolved ### 
Proper disposing of RadioButton control

### Platforms Affected ### 

- UWP

### Testing Procedure ###
Controls gallery

### PR Checklist ###

- [x] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by hartez** [comment]
@yurkinh Do you have a test case which shows this renderer leaking memory because it doesn't unhook the event handlers? Can you describe a situation in which that would happen?

**Comment by hartez** [comment]
Adding a DNM tag until we have some indication that this is necessary.

File: Xamarin.Forms.Platform.UAP/RadioButtonRenderer.cs [hunk 1]
```diff
@@ -209,5 +213,25 @@ void UpdateCheck()
 
 			Control.IsChecked = Element.IsChecked;
 		}
+
+		protected override void Dispose(bool disposing)
+		{
+			if (IsDisposed)
+				return;
+			if (button != null)
```
Comment (hartez) [inline Xamarin.Forms.Platform.UAP/RadioButtonRenderer.cs:221]: This is missing a check for `disposing == true`.

File: Xamarin.Forms.Platform.UAP/RadioButtonRenderer.cs [hunk 2]
```diff
@@ -9,6 +9,10 @@ namespace Xamarin.Forms.Platform.UWP
 	public class RadioButtonRenderer : ViewRenderer<RadioButton, FormsRadioButton>
 	{
 		bool _fontApplied;
+		FormsRadioButton button;
+		PointerEventHandler _pointerPressedHandler;
+
+		public bool IsDisposed { get; private set; }
```
Comment (hartez) [inline Xamarin.Forms.Platform.UAP/RadioButtonRenderer.cs:15]: No need for this to be public; a private `_disposed` member would be sufficient.

File: Xamarin.Forms.Platform.UAP/RadioButtonRenderer.cs [hunk 3]
```diff
@@ -9,6 +9,10 @@ namespace Xamarin.Forms.Platform.UWP
 	public class RadioButtonRenderer : ViewRenderer<RadioButton, FormsRadioButton>
 	{
 		bool _fontApplied;
+		FormsRadioButton button;
```
Comment (hartez) [inline Xamarin.Forms.Platform.UAP/RadioButtonRenderer.cs:12]: No need to add another member; `Control` will be a reference to the Button if it exists.

File: Xamarin.Forms.Platform.UAP/RadioButtonRenderer.cs (hunk 1 above)
Comment (hartez) [inline Xamarin.Forms.Platform.UAP/RadioButtonRenderer.cs:221]: `button` -> `Control`

**Comment by samhouts** [comment]
Since we have an alternate RadioButton implementation in progress, I'll close this one for now. Thank you so much for the work you put into this! :)

**Pull request closed without merging**